.github/workflows/visual-tests.yml

```diff
@@ -0,0 +1,39 @@
+name: Visual tests
+
+on:
+  workflow_dispatch:
+  push:
+  
+env:
+  NODE_VERSION: 16
+
+jobs:
+  build:
+    name: Build and upload visual tests
+    runs-on: ubuntu-latest
+    container:
+      image:  mcr.microsoft.com/playwright:v1.28.1-focal
+    # if: ${{ github.event.workflow_run.conclusion == 'success' || github.event_name == 'workflow_dispatch' }}
```

Comment:
I left it just in case, it is related to the previous point and depends on how and when tests will be launched - we will need this line if `visual tests` worklow will be launched by another workflow (like we have it in `prerelease` already, which is launched only if `tests` passed)